### PR TITLE
feat: WelcomeMessage (DX-2153)

### PR DESCRIPTION
**Pull request by mikaalnaik**
**Fixes or implements VF-XXX**

### Related PRs

- https://github.com/voiceflow/XXXXXXXXX/pull/123

### Checklist

- [ ] Breaking changes have been communicated, including:
    - New required environment variables
    - Renaming of interfaces (API routes, request/response interface, etc)
- [ ] New environment variables have [been deployed](https://www.notion.so/voiceflow/Add-Environment-Variables-be1b0136479f45f1adece7995a7adbfb)
- [ ] Appropriate tests have been written
    - Bug fixes are accompanied by an updated or new test
    - New features are accompanied by a new test

### CODE REVIEW

File: packages/chat/src/components/WelcomeMessage/WelcomeMessage.story.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    description: 'Agent description',
+  },
+};
+
+export const LongContent: Story = {
+  args: {
+    avatar: VF_ICON,
+    title: 'Hello, I am your #1 favourite AI assistant, meant to help you out in all sorts of ways',
+    description:
+      'I can do tons of stuff, like help you with your account, answer questions, and even tell you a joke or two. Just ask me anything!',
```
Comment (gillyb) [inline packages/chat/src/components/WelcomeMessage/WelcomeMessage.story.tsx:34]: love this :)

File: packages/chat/src/components/WelcomeMessage/WelcomeMessage.css.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  textAlign: 'center',
+});
+
+export const avatarContainer = style({
+  marginBottom: 16,
+});
+
+export const welcomeMessageTitle = style({
+  margin: '0 0 4px 0',
+  fontSize: '22px',
```
Comment (ninabondar) [inline packages/chat/src/components/WelcomeMessage/WelcomeMessage.css.ts:21]: are sizes not tokenized?

File: packages/chat/src/components/WelcomeMessage/WelcomeMessage.css.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  fontWeight: 700,
+  color: COLORS.NEUTRAL_DARK[900],
+});
+
+export const welcomeMessageDescription = style({
+  margin: 0,
+  fontSize: '14px',
```
Comment (ninabondar) [inline packages/chat/src/components/WelcomeMessage/WelcomeMessage.css.ts:28]: same q

File: packages/chat/src/components/WelcomeMessage/WelcomeMessage.css.ts [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  padding: '48px 20px 24px 20px',
+  textAlign: 'center',
+});
+
+export const avatarContainer = style({
+  marginBottom: 16,
+});
+
+export const welcomeMessageTitle = style({
+  margin: '0 0 4px 0',
```
Comment (gillyb) [inline packages/chat/src/components/WelcomeMessage/WelcomeMessage.css.ts:20]: why not just `marginBottom: 4px` ?

**Review by gillyb — APPROVED** [review]
LFG

**Review by ninabondar — APPROVED** [review]

File: apps/documentation/src/pages/atomic/welcome-message/index.mdx [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+# WelcomeMessage
+
+## Base
+
+<StoryEmbed for="WelcomeMessage" name="Base" />
+
+## Active States
+
+#
```
Comment (ninabondar) [inline apps/documentation/src/pages/atomic/welcome-message/index.mdx:12]: I think it's redundant

**Pull request merged**